home/home.nix

```diff
@@ -57,14 +59,16 @@
         "Terminus"
       ];
     })
-    twitter-color-emoji
     noto-fonts-emoji
     powerline-fonts
+    twitter-color-emoji
+
 
     # Node
     nodejs
 
     # Python
+    pipenv # Environment management
```

Comment:
I'm not sure what way to manage python tooling, should it be installed within the Python 3.10 environment?

Reply:
I don't believe pipenv is available via `python310Packages`

https://search.nixos.org/packages

https://github.com/NixOS/nixpkgs/tree/master/pkgs/development/python-modules/

Some Python apps appear to be available via both `pkgs` and `python310Packages`. `black` is one example:

https://search.nixos.org/packages?channel=22.11&query=black